Tests/Foundation/Tests/TestFileHandle+Async.swift

```diff
@@ -0,0 +1,426 @@
+        // € shares the same prefix bit mask as _SEPERATOR_PREFIX
+        try await _test_lines(with: String(data: Data([0x61, 0xE2, 0x82, 0xAC, 0x62, 0xE2, 0x82, 0xAC, 0x63, 0xE2, 0x82, 0xAC]), encoding: .utf8)!)
+    }
+    
+    func test_seperator_prefix_end() async {
+        let lines = await [UInt8(0x48), UInt8(0xE2)].async.lines.collect()
+        XCTAssertEqual(lines, ["H\u{FFFD}"])
+    }
+    
+    func test_seperator_prefix_and_continuation_end() async {
+        let lines = await [UInt8(0x48), UInt8(0xE2), UInt8(0x80)].async.lines.collect()
+        XCTAssertEqual(lines, ["H\u{FFFD}"])
+    }
+    
+    func test_seperator_prefix_and_continuation_without_fin() async {
+        let lines = await [UInt8(0x48), UInt8(0xE2), UInt8(0x80), UInt8(0x48)].async.lines.collect()
+        XCTAssertEqual(lines, ["H\u{FFFD}H"])
+    }
+    
+    static var allTests : [(String, (TestFileHandleAsync) -> () async throws -> ())] {
+        var tests: [(String, (TestFileHandleAsync) -> () async throws -> ())] = [
+            ("testLines", test_lines),
+            ("testLargeFileLines", test_large_file_lines),
+            ("testEmptyFileLines", test_empty_file_lines),
+            ("testAllNewlines", test_all_newlines),
+            ("testPartialNEL", test_partial_NEL),
+            ("testTrailingPartialNEL", test_trailing_partial_NEL),
+            ("testTrailingCR", test_trailing_cr),
+            ("testTrailingLF", test_trailing_lf),
+            ("testCRCRLF", test_crcrlf),
+            ("testCharacters", test_characters),
+            ("testScalars", test_scalars),
+            ("testUTF8Bytes", test_utf8_bytes),
+            ("testExtraTrailingContinuationBytesScalars", test_extra_trailing_continuation_bytes_scalars),
+            ("testExtraContinuationBytesScalars", test_extra_continuation_bytes_scalars),
+            ("testMultiByteAfterContinuation", test_multibyte_after_continuation),
+            ("testEmptyFileCharacters", test_empty_file_characters()),
+            ("testEmptyFileScalars", test_empty_file_scalars),
+            ("testEmptyFileUTF8Bytes", test_empty_file_utf8_bytes()),
+            ("testNullDevice", test_nullDevice),
+            ("testPipe", test_pipe),
+            ("testPipeUnexpectedClose", test_pipe_unexpected_close),
+            ("testDevice", test_device),
+            ("testNELPrefixButNoSuffix", test_nel_prefix_but_no_suffix),
+            ("testNELPrefixEnd", test_nel_prefix_end),
+            ("testNELPrefixAndSuffix", test_nel_prefix_and_suffix),
+            ("testSeparatorPrefixAndContinuationEnd", test_seperator_prefix_and_continuation_end),
+            ("testSeparatorPrefixButNoSuffix", test_seperator_prefix_but_no_suffix),
+            ("testSeparatorPrefixAndContinuationEnd", test_seperator_prefix_and_continuation_end),
+            ("testSeparatorPrefixAndContinuationWithoutFin", test_seperator_prefix_and_continuation_without_fin)
+        ]
+        
+        return tests
```

Comment:
@Catfish-Man Is this what you meant here? This method was missing its termination.